src/components/Nav/Nav.js

```diff
@@ -1,7 +1,182 @@
-import React from 'react';
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { SIDEBAR_INSIDE } from './SIDEBAR_INSIDE';
+import SideInside from './SideInside';
 
 const Nav = () => {
-  return <div>hi</div>;
+  const [displayShow, setDisplayShow] = useState(false);
+
+  const showSidebar = () => {
+    setDisplayShow(!displayShow);
```

Comment:
```suggestion
    setDisplayShow(prev => !prev);
```

이전값을 이용해서 새로운 값을 만들때에는 함수형 업데이트를 사용하시는게 좀 더 안전합니다!

Reply:
네 리뷰 반영하겠습니다~